src/chrono/timestepper/ChTimestepperHHT.h

```diff
@@ -122,6 +125,22 @@ class ChApi ChTimestepperHHT : public ChTimestepperIIorder, public ChImplicitIte
     virtual void Advance(const double dt  ///< timestep to advance
                          ) override;
 
+    /// Get an indicator to tell whether the iteration in current step tends to convergence or divergence.
+    /// This could be helpful if you want to fall back to iterate again via using more rigorous stepper settings, 
+    /// such as smaller fixed time stepper, turning off ModifiedNerton,etc, 
+    /// WHEN you are not satisfied by current iteration result.
+    bool GetConvergenceFlag() { return convergence_trend_flag; };
+
+    /// Set the threshold of norm of R, which is used to judge the trend of convergency.
+    /// For different systems, this threshold may be much different, such as a mini robot and a huge wind turbine.
+    /// You could turn on 'verbose' of HHT stepper and look at the norm of R for your system firstly, 
+    /// and then set a suitable threshold of norm of R. 
+    /// This threshold should always be larger than the normal norm of R for your system.
+    void SetThreshold_R(double mv) { threshold_R = mv; };
```

Comment:
Please change to:
void SetThreshold_R(double mv) { threshold_R = mv; }

Reply:
See commit 9e37f48 please.